arch/configure.defaults

```diff
@@ -119,8 +119,9 @@ FCDEBUG         =       # -g $(FCNOOPT)
 FORMAT_FIXED    =       -ffixed-form
 FORMAT_FREE     =       -ffree-form -ffree-line-length-huge
 FCSUFFIX        =
+FCCOMPAT        =
 BYTESWAPIO      =       -fendian=big
-FCBASEOPTS_NO_G =       -Wno=101,139,155,158 $(FORMAT_FREE) $(BYTESWAPIO)
+FCBASEOPTS_NO_G =       -Wno=101,139,155,158 $(FORMAT_FREE) $(BYTESWAPIO) $(FCCOMPAT)
```

Comment:
This build target is for the `g95` compiler if I'm not mistaken. Since the `g95` compiler hasn't been actively developed in quite some time, I'd suggest just leaving this configuration stanza as-is.